README.md

```diff
@@ -1 +1 @@
-# gitflow
\ No newline at end of file
+# gitflow
```

Comment:
One of the most important parts of any project is the README file, be sure your README file is descriptive and complete according to your project, take into consideration that not always you should use all of the sections provided by the [Template of Microverse](https://github.com/microverseinc/readme-template), it depends of the projects.

- [ ] Use the [Template of Microverse for README files](https://github.com/microverseinc/readme-template) and fill it with your information about you and your project.

Reply:
Thank you. I am still trying to get used to the new system. Thank you again. Let me work on it
I hope it doesn't affect the number of reviews I am left with

Reply:
I managed to add one extra line in the README.md file. Is it fine or I add the other sections?